domain/build.gradle

```diff
@@ -68,6 +70,9 @@ dependencies {
       'org.robolectric:robolectric:4.3',
       project(":testing"),
   )
+  androidTestImplementation(
```

Comment:
We don’t use instrumentation tests for domain layer tests.

Reply:
Removed.